javascript/javascript2/week1/findTheShortestWord.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Hack Your Future, Javascript2, homework Week1
+ * this file includes:
+    Find the shortest word function
+    Find and count the Danish letters function
+
+*/
+
+// Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function wordFinder(list) {
+    let shortestWord = list[0];
```

Comment:
The function `wordFinder()` assumes that `list` will have at least one element. That is true in this example but is a case we should think about.

Reply:
Yes. got it. Thanks for the comment.